src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java

```diff
@@ -65,7 +65,7 @@
 //                          assertThat(find.commandContext()).isEqualTo(commandContext);
```

Comment:
why is this file commented out

Reply:
I don't know, but looks like you commented it out? `git blame src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java` shows:

```
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 309) //                          assertThat(
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 310) //                                  find.logicalExpression()
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 311) //                                      .comparisonExpressions
...
```

Reply:
For:

```
commit 52e3de28808f2c3b756ac2c58e11233a0dee7838 (HEAD)
Author: YuqiDu <istimdu@gmail.com>
Date:   Mon Oct 23 10:56:01 2023 -0700

    Support and or (#565)
```

Maybe accidentally commented out?

Reply:
sorry, fixed!

Reply:
also fixed in cql branch